src/pombot/commands/start.js

```diff
@@ -7,5 +7,22 @@ export default createCommand({
   name: 'start',
   description: 'Starts a pom timer.',
 }, () => {
-  return `> Theoretically I should be starting a pom right now.`;
+  // get pom status
+  const pomStatus = 'running';
+
+  switch (pomStatus) {
```

Comment:
i added this but i'm not sure why.

what i'm trying to figure out now is how to keep the timer to the current conversation. if i start a pom in a channel, and try to start one in dm, the bot says one is already going on